secrets/secrets-kubernetes/Makefile

```
@@ -0,0 +1,12 @@
+VERSION?=dev
+IMG?=ghcr.io/wasmcloud/wasmcloud-contrib/secrets-kubernetes:$(VERSION)
```

It's not yet clear where we'll end up publishing these, but this is the format I'd used for the Vault backend: https://github.com/wasmCloud/wasmCloud-contrib/blob/main/.github/workflows/secrets-vault.yml#L16
```suggestion
IMG?=ghcr.io/wasmcloud/contrib/secrets-kubernetes:$(VERSION)
```

---

makes sense! also updating kustomize